experimental/dds/tree2/src/test/shared-tree/fuzz/fuzzEditGenerators.ts

```diff
@@ -346,6 +361,63 @@ export const makeTransactionEditGenerator = (
 	};
 };
 
+export const makeDisconnectGenerator = (): Generator<DisconnectOp, EditState> => {
+	return (state) => ({
+		type: "disconnect",
+		index: state.treeIndex,
+		isObserver: state.treeIndex < 3,
+	});
+};
+
+export const makeReconnectGenerator = (): Generator<ReconnectOp, EditState> => {
+	return (state) => ({
+		type: "reconnect",
+		index: state.treeIndex,
+		isObserver: state.treeIndex < 3,
+	});
+};
+
+export function makeConnectionOpGenerator(
+	opWeights: Partial<EditGeneratorOpWeights>,
+): Generator<ConnectionOps, EditState> {
+	const passedOpWeights = {
+		...defaultEditGeneratorOpWeights,
+		...opWeights,
+	};
+	const connectionOpGenerator = createWeightedGenerator<ConnectionOps, EditState>([
+		[
+			makeDisconnectGenerator(),
+			passedOpWeights.disconnect,
+			({ containersInfo, treeIndex }) => {
+				const containersExist = containersInfo !== undefined;
+				if (!containersExist) {
+					return false;
+				}
+				return containersExist;
```

Comment:
This is unnecessarily confusing. Can just `return containersInfo !== undefined`.

Reply:
simplified to `return containersInfo !== undefined`